blogpost/post.md

```diff
@@ -0,0 +1,261 @@
+# Compiling NumPy code into C++ or CUDA via `torch.compile`
+
+Quansight engineers have implemented support for tracing through NumPy code via
+`torch.compile` in PyTorch 2.1.
+This feature leverages PyTorch's compiler to generate efficient fused
+vectorized code without having to modify your original code. Even more, it
+also allows for executing NumPy functions on CUDA just by running them through
+`torch.compile` under `torch.device("cuda")`!
+
+In this post, we go over how to use this feature and give a few tips and tricks
+to make the most out of it.
+
+
+## Compiling NumPy code into Parallel C++
+
+We take as our running example one step in a K-Means algorithm.
+This piece of code is borrowed from this [NumPy book](https://realpython.com/numpy-array-programming/#clustering-algorithms)
```

Comment:
From the hair splitter gallery: the NumPy book is what Travis wrote back in the day, later to become the basis of the current numpy docs (I think)